docs/flush-tables-with-read-lock.md

```diff
@@ -56,25 +57,22 @@ used xtrabackup will wait on `UPDATE/ALTER/REPLACE/INSERT` queries to
 finish.
 
 The time needed for a specific query to complete is hard to predict. We assume that the long-running queries will not finish in a timely manner. Other queries which run for a short time finish quickly. xtrabackup uses the value of
-–ftwrl-wait-threshold option to specify the long-running queries
-and will block a global lock. In order to use this option
-xtrabackup user should have `PROCESS` and `SUPER` privileges.
+`–ftwrl-wait-threshold option to specify the long-running queries
+and will block a global lock. To use this option
+xtrabackup user should have `PROCESS` and `CONNECTION_ADMIN` privileges.
 
 ## Kill the blocking queries
 
 The second option is to kill all the queries which prevent from acquiring the
-global lock. In this case, all queries which run longer than `FLUSH TABLES WITH
-READ LOCK` are potential blockers. Although all queries can be killed,
+global lock. In this case, all queries which run longer than `FLUSH TABLES WITH READ LOCK` are potential blockers. Although all queries can be killed,
 additional time can be specified for the short running queries to finish using
 the `--kill-long-queries-timeout` option. This option
 specifies the time for queries to complete, after the value is reached, all the
```

Comment:
future tense?
This option specifies a query time limit. After the specified time is reached, the server kills the query.

Reply:
fixed